api/schema/application_aws_resources_resource.go

```diff
@@ -0,0 +1,185 @@
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 eips to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_flowlogs": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 flow logs to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_instances": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 instances to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_instanceconnectendpoints": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 instance connect endpoints to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_internetgateways": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 internet gateways to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_natgateways": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 nat gateways to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_networkacls": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 network acls to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_networkinterfaces": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 network interfaces to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_routetabless": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 route tabless to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_securitygroups": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 security groups to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_securitygrouprules": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 security group rules to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_spotfleetrequests": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 spot fleet requests to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_spotinstancerequests": resource_schema.SetAttribute{
+					ElementType: types.StringType,
+					Description: "IDs of AWS ec2 spot instance requests to associate with Cloudsecure Application.",
+					Optional:    true,
+				},
+				"aws_ec2_subnets": resource_schema.SetAttribute{
```

Comment:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/subnet
```suggestion
				"aws_subnets": resource_schema.SetAttribute{
```

Reply:
Please fix all other attribute names below to match the name of the corresponding AWS Terraform resource, and suffix with `_ids`.